src/amo/components/HomepageShelves/index.js

```diff
@@ -76,13 +75,10 @@ export const HomepageShelvesBase = (props: InternalProps): React.Node => {
           ? INSTALL_SOURCE_FEATURED_COLLECTION
           : INSTALL_SOURCE_FEATURED;
 
-      const hasThemes =
```

Comment:
Because you use this logic twice below, once for `count` and once for `isTheme`, it might make sense to keep this `hasThemes` `const` and just use it in both of those places.